test/Libraries/GeometryColorTests/GeometryColorTests.csproj

```diff
@@ -96,7 +90,12 @@
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="packages.config" />
+    <PackageReference Include="DynamoVisualProgramming.LibG_227_0_0">
+      <Version>*</Version>
```

Comment:
Does this wildcard mean whatever is latest?

Reply:
Yes, that's correct